pkg/glance/pvc.go

```diff
@@ -0,0 +1,33 @@
+package glance
+
+import (
+	"context"
+	"fmt"
+
+	glancev1beta1 "github.com/openstack-k8s-operators/glance-operator/api/v1beta1"
+	"github.com/openstack-k8s-operators/lib-common/pkg/common"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+// Pvc - creates and returns a PVC object for a backing store
+func Pvc(ctx context.Context, r common.ReconcilerCommon, api *glancev1beta1.GlanceAPI, labels map[string]string) (*corev1.PersistentVolumeClaim, error) {
```

Comment:
We should add generic functions to lib-common, like we have for deployment [1] that you can use it like pvc := NewPVC(pvc, p2, p3) . Then pvc.CreateOrPatch(), pvc.Delete(), ... . As this is required by multiple services would be good to move this to the new way to create/manage resources. In here we'd then add the raw PVC for glance.

[1] https://github.com/openstack-k8s-operators/lib-common/blob/master/pkg/common/deployment.go#L34